.github/workflows/crypto-bigint.yml

```
@@ -142,7 +147,24 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v3
-      - uses: dtolnay/rust-toolchain@master
+      - uses: risc0/risc0/.github/actions/rustup@main
         with:
           toolchain: stable
       - run: cargo doc --all-features
+
+  risczero-test:
+    name: Risc Zero Test
+    runs-on: [self-hosted, prod, Linux, cpu]
+    steps:
+      - uses: actions/checkout@v3
+      - uses: actions/checkout@v3
+        with:
+          repository: risc0/risc0
+          path: risc0
+      - uses: risc0/risc0/.github/actions/rustup@main
+      # install the risc0 toolchain using the latest release of cargo-risczero
+      - run: cargo install cargo-risczero
+      - run: cargo risczero install
+      # rebuild with the latest cargo-risczero with the experimental feature
+      - run: cargo install cargo-risczero --path risc0/risc0/cargo-risczero --features experimental
+      - run: cargo risczero test -- --release --all-features
```

```suggestion
      - run: cargo install cargo-risczero --path risc0/risc0/cargo-risczero --features experimental
      - run: cargo risczero install
      - run: cargo risczero test -- --release --all-features
```

Building twice will be pretty slow